openfisca_web_api_preview/app.py

```diff
@@ -44,6 +46,16 @@ def get_variable(id):
             raise abort(404)
         return jsonify(variable)
 
+    @app.route('/swagger')
+    def root():
+        file = open(OPEN_API_CONFIG_FILE, 'r')
+        spec = yaml.load(file)
+        country_package_name = data['country_package_metadata']['name'].title()
+        spec['info']['title'] = spec['info']['title'].replace("{CONTRY_PACKAGE_NAME}", country_package_name)
+        spec['info']['description'] = spec['info']['description'].replace("{CONTRY_PACKAGE_NAME}", country_package_name)
+        spec['host'] = os.environ.get('SERVER_NAME')
+        return jsonify(spec)
```

Comment:
Shouldn't this be cached on startup? Here, it seems we're re-reading the file from disk on each request.